rllib/examples/env/recommender_system_envs_with_recsim.py

```diff
@@ -94,7 +92,7 @@ def __init__(self, doc_id, features, video_length=None, quality=None):
 
 
 def iev_document_sampler_creator(env_ctx):
-    return iev.IEvVideoSampler(doc_ctor=SingleClusterIEvVideo, seed=env_ctx["seed"])
+    return iev.UtilityModelVideoSampler(doc_ctor=iev.IEvVideo, seed=env_ctx["seed"])
```

Comment:
if you meant to revert these back, you can delete the SingleClusterIEvVideo class above.
I'd be interested in seeing the obs vectors, and understand what our documents look like ...

Reply:
Just wanted to make things comparable again between RecSim's agent and ours.